packages/esm-patient-registration-app/src/patient-registration/patient-registration.scss

```diff
@@ -28,8 +28,9 @@
     margin: $spacing-05 0 $spacing-05 0;
 }
 
-.fixedPosition {
+.fixedColumn {
     position: fixed;
```

Comment:
Should not be position fixed. Please remove.

Reply:
Sure!

Reply:
@FlorianRappl the left column needs to be `position:fixed`, because the right column, i.e. the form is scrollable and the left column is kind of navigator for sections on the right form.
![image](https://user-images.githubusercontent.com/51502471/123624550-50c23e80-d82c-11eb-8d04-0d84b8dded92.png)

Reply:
I don't think it does.

1. There is `position: sticky`
2. You can give the outer container full height,but limited to full height (i.e., not scrollable) - then have the right box full the height with an auto scrollable

This way there is no position fixed and it just works.

Reply:
> There is position: sticky
I just tried applying position sticky, but it didn't work. 
Btw, `position: fixed` was already there in the code, I didn't add it.
> You can give the outer container full height,but limited to full height (i.e., not scrollable) - then have the right box full the height with an auto scrollable

In this way, it will give a scrollbar to the right container, which is not expanded to full width and won't that look bad?
![image](https://user-images.githubusercontent.com/51502471/123631381-52900000-d834-11eb-9bab-5f9e8cfba430.png)
As you can see in the image, the outer container of the right column is only limited to the grey part. What I can do is expand it to full width. Do you want me to go with this?

Reply:
> Btw, position: fixed was already there in the code, I didn't add it.

Tell me something I don't know :)

Reply:
Sorry - the gray box you see is not the right element I refer to. I always refer to logical elements, not physical ones...

So when I talk about the right box I mean the container that hosts the gray box. And, of course, this container takes the remaining space (vertical and horizontal) from the outer container. Therefore, for me, there was no question about whether or not take the full width. It will always be taken.

Reply:
Ohh okay, got it.  I'll do it.
Thanks!

Reply:
https://user-images.githubusercontent.com/51502471/123639650-0b0e7180-d83e-11eb-982e-60f589b5d9ba.mp4

Reply:
The blank space above is of the navigation bar, which is not visible due to some error.
Is this good @FlorianRappl ?

Reply:
Hm, in the video I don't see a real scrollbar. The scrollbar I see is not scrolling and is at 100%.

Reply:
![image](https://user-images.githubusercontent.com/51502471/123642417-e667c900-d840-11eb-9598-120121d665bf.png)
This is the scrollbar. Actually `Grid` is being used  which doesn't take up the full width in the desktop view and this is what I asked that time, to extend it to full width,

What I can do is remove the Grid and add my own flex to the container. What you say about it?

Reply:
It wasn't looking good and hence I removed the scrollbar.

Reply:
> What I can do is remove the Grid and add my own flex to the container. What you say about it?

Sounds good to me.

Reply:
> It wasn't looking good and hence I removed the scrollbar.

There should be just one scrollbar - and it should look as expected (i.e., if there an overflow the scrollbar should indicate that).

Reply:
Got it! I'll do it.
Thanks!